bootstrap_cfn/stacks/iam-deploy.json

```diff
@@ -0,0 +1,74 @@
+{
+    "CrossAccountRole" : {
+        "Type" : "AWS::IAM::Role",
+        "Properties" : {
+            "AssumeRolePolicyDocument" : {
+                "Statement" : [
+                     {
+                         "Effect" : "Allow",
+                         "Principal" : {
+                             "AWS": "arn:aws:iam::ACCOUNT_NUMBER_HERE:root"
```

Comment:
Hmm, that would make the code cleaner. To be clear you'd update the paramaters section from the code and just Ref it in the IAM section?